src/native/clr/include/constants.hh

```diff
@@ -0,0 +1,143 @@
+#pragma once
+
+#include <fcntl.h>
+#include <sys/system_properties.h>
+
+#include <string_view>
+
+#include "shared/cpp-util.hh"
+
+namespace xamarin::android {
+	class Constants
+	{
+#if INTPTR_MAX == INT64_MAX
```

Comment:
I'm not sure we need to care about this at this time. CoreCLR only supports 64-bit ABIs on Linux, and this is unlikely to change.

Reply:
Au contraire. The 32-bit support is nearly working except for corehost build (which is not used on Android anyway), for both x86 and arm32.

Reply:
Yeah, it wasn't working when I started to work on #9572 but the goal is that it'll work, so we should always keep 32-bit runtimes in mind here.